tests/aritmeticops.py.uast

```diff
@@ -8,6 +8,10 @@ Module {
 .  .  Line: 1
 .  .  Col: 1
 .  }
+.  Properties: {
```

Comment:
This should be `EndPosition` instead of `Properties`, but I guess that's something still to be done in the normalization pass, right?